docs/scripts/generate_skills_docs.py

```
@@ -0,0 +1,144 @@
+    description: str
+    github_url: str
+    category: str = "General"
+    emoji: str = "🤖"
+
+
+def _fetch_text(url: str) -> Optional[str]:
+    try:
+        resp = requests.get(url, timeout=10)
+        if resp.status_code == 200:
+            return resp.text
+    except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)